packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -235,6 +233,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/webpack
 
 %changelog
+* Mon Nov 23 2020 Evgeni Golov - 4.0.0-0.2.pre.master
```

Comment:
not sure why but linter seems unhappy about this

Reply:
ignore me, you were too fast again

Reply:
not anymore ;)